pb/api/v1/data.proto

```diff
@@ -184,13 +189,20 @@ message GetTabSummaryRequest{
 
   // Name of the particular tab to fetch the summary for.
   string tab = 3;
+
+  // Number of top failings tests to fetch.
+  // At most this many failing tests will be returned as part of TabSummary.
+  // Will be displayed in summarized view (failing tests summary).  
+  int32 num_failing_tests = 4;
```

Comment:
nit: Seeing this again, wonder if this should be more aligned with 'page size' or similar arguments (combined with a default sort by failure rate, that we can adjust later). e.g. https://cloud.google.com/apis/design/design_patterns#list_pagination?

Reason I was thinking about this is because of https://github.com/googleapis/googleapis/blob/master/google/devtools/resultstore/v2/resultstore_download.proto#L378; ExportInvocation for instance grabs the resource and subresources for a particular invocation, but paginated in case of returning a _lot_ of data. Not sure this works well with GetTabSummaryRequest or ListTabSummariesRequest, but possibly worth considering?

Reply:
I thought that detailed data was slated to go into new endpoints, ListTabFailures and ListTabHealthiness.

Summarized data to be displayed to the UI could be put here for convenience's sake, but we don't want a complex system for asking for "X" failures _here_, we want it in ListTabFailures. ListTabSummaries (and GetTabSummary) should only contain directly relevant information, with maybe link(s) to an API that will tell you more.

Reply:
@chases2 is right, the plan is to include pagination, sorting and filtering for new endpoints which will provide detailed data on failing tests & healthiness info.

The  reason is that I didn't want to hardcode the # of most failing / flaky tests in the summary view.
I guess we can just define a constant say 5 and the FailuresSummary always display up to 5 tests (i.e. the most important data at a glance)

Reply:
@michelle192837 @chases2 - I removed the field in the request and for now set the default # of top failing tests to return as part of the summary. PTAL - thanks!